pinc/misc.inc

```diff
@@ -800,13 +800,15 @@ function get_upload_err_msg($upload_error_code)
 // Represent $n bytes as a  $m kB-- $m PB string
 function humanize_bytes($n)
 {
-    $fmt   = "%d B";
-    $units = array("PB", "TB", "GB", "MB", "kB");
-    while($n >= 1024) {
-        $n   /= 1024.0;
-        $fmt  = "%.2f " . array_pop($units);
+    $i = 0;
+    $units = array("B", "KiB", "MiB", "GiB", "TiB", "PiB");
```

Comment:
https://ux.stackexchange.com/questions/13815/files-size-units-kib-vs-kb-vs-kb wraps up my thoughts on `KB` vs `KiB`, mostly that while it is technically more accurate, I suspect it's mostly going to just confuse people. Both Mac and Windows represent files in the former way and I think we should too. (that does mean fixing `kB` to `KB` however). I do prefer your updated logic and dropping the decimal if it's zero.

Reply:
Fair enough, I have been using a Linux system recently since my windows computer broke down and it uses KiB etc. I'll change it.

Reply:
Thanks, and welcome to all-Linux land!